src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_layout.scss

```diff
@@ -56,20 +56,6 @@ depends on a lot of factors:
 </div>
 ```
 */
-.main-header {
-    @include rem(line-height, $logo-height);
-    position: relative;
-
-    @media print {
-        display: none;
-    }
-}
-
-.main-header-logo {
-    height: $logo-height;
-    vertical-align: bottom;
-}
```

Comment:
This code had been moved to a separate file in master. see https://github.com/liqd/adhocracy3/pull/2757#issuecomment-251354210.